apps/console/src/components/pages/protected/trust-center/documents/create-document.sheet.tsx

```diff
@@ -0,0 +1,320 @@
+                Delete
+              </Button>
+
+              <ConfirmationDialog
+                open={isDeleteDialogOpen}
+                onOpenChange={setIsDeleteDialogOpen}
+                onConfirm={handleDeleteConfirm}
+                title="Delete Document"
+                description={
+                  <>
+                    Are you sure you want to delete this document?
+                    <br />
+                    This action cannot be undone.
+                  </>
+                }
+                confirmationText="Delete"
+                confirmationTextVariant="destructive"
+                showInput
+              />
+            </>
+          )}
+        </SheetHeader>
+
+        <form onSubmit={handleSubmit(onSubmit)} className="mt-6 space-y-5">
+          {/* Title */}
+          <div>
+            <Label>Title</Label>
+            <Input placeholder="Document title" {...register('title')} />
+            {errors.title && <p className="text-red-500 text-sm mt-1">{errors.title.message}</p>}
+          </div>
+
+          {/* Category */}
+          <div>
+            <Label>Category</Label>
+            <Input placeholder="Category" {...register('category')} />
+            {errors.category && <p className="text-red-500 text-sm mt-1">{errors.category.message}</p>}
+          </div>
+
+          {/* Visibility */}
+          <div>
+            <Label>Visibility</Label>
+            <Select
+              onValueChange={(val) =>
+                setValue('visibility', val as TrustCenterDocTrustCenterDocumentVisibility, {
+                  shouldValidate: true,
+                })
+              }
+              value={visibilityValue || ''}
+            >
+              <SelectTrigger>
+                <SelectValue placeholder="Select visibility" />
+              </SelectTrigger>
+              <SelectContent>
+                {enumToOptions(TrustCenterDocTrustCenterDocumentVisibility).map(({ value, label }) => (
+                  <SelectItem key={value} value={value}>
+                    {label}
+                  </SelectItem>
+                ))}
+              </SelectContent>
+            </Select>
+            {errors.visibility && <p className="text-red-500 text-sm mt-1">{errors.visibility.message}</p>}
+          </div>
+
+          {/* Tags */}
+          <div>
+            <Label>Tags</Label>
+            <Controller
+              control={control}
+              name="tags"
+              render={({ field }) => (
+                <MultipleSelector creatable value={(field.value ?? []).map((tag) => ({ value: tag, label: tag }))} onChange={(selected) => field.onChange(selected.map((s) => s.value))} />
+              )}
+            />
+            {errors.tags && <p className="text-red-500 text-sm mt-1">{errors.tags.message as string}</p>}
+          </div>
+
+          {/* File Upload + Existing File */}
+          <div>
+            <Label>PDF File</Label>
```

Comment:
```suggestion
            <Label>Document file</Label>
```

Can you add helper text
> Upload the final report or certification for customers to view in your Trust Center